public/app/features/alerting/unified/components/contact-points/__mocks__/alertmanager.config.mock.json

```diff
@@ -1,9 +1,9 @@
 {
   "template_files": {
-    "slack-template": "{{ define 'slack-template' }} Custom slack template {{ end }}",
-    "custom-email": "{{ define 'custom-email' }}  Custom email template {{ end }}",
-    "provisioned-template": "{{ define 'provisioned-template' }}  Custom provisioned template {{ end }}",
-    "template with spaces": "{{ define 'template with spaces' }}  Custom template with spaces in the name {{ end }}"
+    "slack-template": "{{ define \"slack-template\" }} Custom slack template {{ end }}",
```

Comment:
I needed to update thes edefines for the test. These values were not valid. define must use doble quotes.